src/Compiler/Driver/GraphChecking/FileContentMapping.fs

```diff
@@ -56,7 +56,7 @@ let visitSynModuleDecl (decl: SynModuleDecl) : FileContentEntry list =
         | SynModuleDecl.NestedModule(moduleInfo = SynComponentInfo(longId = [ ident ]; attributes = attributes); decls = decls) ->
             yield! visitSynAttributes attributes
             yield FileContentEntry.NestedModule(ident.idText, List.collect visitSynModuleDecl decls)
-        | SynModuleDecl.NestedModule _ -> failwith "A nested module cannot have multiple identifiers"
+        | SynModuleDecl.NestedModule _ -> () // "A nested module cannot have multiple identifiers"
```

Comment:
If you are leaving in this comment (which I think is good), maybe also clarify that this already causes an error in parsing that will be reported (if I understood correctly)?

Reply:
Yep. Added.